cypress/e2e/home.cy.ts

```diff
@@ -33,9 +36,37 @@ describe('Test Movapp - Home page', () => {
     checkVisibility('h2', 'Ukrajinská abeceda');
   });
 
-  it('Test of footer of the page', () => {
+  it('Test of languages of the page', () => {
     cy.contains('Movapp mluví dalšími jazyky').should('be.visible');
     cy.contains('Slovensky').should('be.visible');
     cy.contains('Polski').should('be.visible');
   });
-});
+
+  it('Test of the footer of the page', function(){
+    cy.contains('facebook');
+    cy.xpath ("//a[contains(@href, 'facebook')]").click();
+    cy.contains('instagram');
+    cy.xpath ("//a[contains(@href, 'instagram')]").click();
+    cy.contains('twitter');
+    cy.xpath ("//a[contains(@href, 'twitter')]").click();
+    cy.contains('linkedin');
+    cy.xpath ("//a[contains(@href, 'linkedin')]").click();
+    cy.contains('telegram');
+    cy.xpath ("//a[contains(@href, 't.me')]").click();
+
+  
+    cy.xpath ("//a[contains(@href, 'pomahejukrajine')]").click();
+    cy.xpath ("//a[contains(@href, 'stojimezaukrajinou')]").click();
+    cy.xpath ("//a[contains(@href, 'cesko.digital')]").click();
+
+    cy.contains('footer button', 'Našli jste chybu nebo máte návrh na zlepšení?').invoke('text').should('equal', 'Našli jste chybu nebo máte návrh na zlepšení?');
+
+    cy.xpath ("//a[contains(@href, 'github')]").click();
+    cy.xpath ("//a[contains(@href, 'creativecommons')]").click();
```

Comment:
No need for xpath, can be achieved with core functions:

cy.contains('CC BY-NC 4.0 Mezinárodní').should('have.attr', 'href').and('include', 'creativecommons');

Reply:
I used XPath because of its ability to select selectors very precisely. If it's not necessary to select elements with such precision, then that's okay.

Reply:
I also don't see value in Xpath right now, you can achieve the same results with native Cypress syntax without an extra dependency. Cypress syntax also seems more approachable to beginners, making it easier for other people to add Cypress tests. We try to optimize for readability and beginner-friendliness with our libraries choices :)